packages/react/src/components/ScrollablePane/ScrollablePane.test.tsx

```diff
@@ -4,6 +4,10 @@ import { ScrollablePane } from './ScrollablePane';
 
 describe('ScrollablePane', () => {
   it('renders correctly', () => {
+    // Trying to call MutationObserver.observe in ScrollablePane.componentDidMount on a fake node
+    // causes an exception in React 17. Just mock it since it's not important for this test.
+    jest.spyOn(MutationObserver.prototype, 'observe').mockImplementation();
```

Comment:
**suggestion:** I'd throw an exception inside to mock so we are 100% sure it's not being used and if it will by some accident, contributor will be notified.

```
jest.spyOn(MutationObserver.prototype, 'observe').mockImplementation(()=>{
  throw new Error('those are not the droids you're looking for')
})
```

Reply:
but the whole point of this mock is that it **is** being used. so we shouldn't be throwing an error